zap/src/main/java/org/zaproxy/zap/view/MultipleRegexesOptionsPanel.java

```diff
@@ -212,8 +226,12 @@ protected void init() {
         @Override
         protected boolean validateFields() {
             try {
-                Pattern.compile(getRegexTextField().getText(), Pattern.CASE_INSENSITIVE);
-            } catch (PatternSyntaxException e) {
+                String trimmedRegex = getRegexTextField().getText().trim();
+                if (regexes != null && regexes.contains(trimmedRegex)) {
+                    throw new IllegalArgumentException(TEXT_ALREADY_EXIST_INVALID_REGEX_DIALOG);
```

Comment:
- We should not use exceptions for normal/common control flow (when we can avoid it, they are expensive to create). This should show the dialogue and return.
 -  I'd suggest using other title (e.g. `Duplicated Regular Expression`) for the error and show without `The provided regular expression is not valid:`, seems unnecessary.